sql/moz-fx-data-shared-prod/telemetry_derived/mobile_engagement_clients_v1/query.sql

```diff
@@ -0,0 +1,197 @@
+    locale,
+    country,
+    isp,
+    is_dau, --will work once the column is added by Kik via PR#5434
+    is_wau, --will work once the column is added by Kik via PR#5434
+    is_mau --will work once the column is added by Kik via PR#5434
+  FROM
+    `moz-fx-data-shared-prod.klar_android.baseline_clients_last_seen`
+  WHERE
+    submission_date = @submission_date
+  UNION ALL
+  --Klar iOS
+  SELECT
+    'Klar iOS' AS source,
+    sample_id,
+    submission_date,
+    client_id,
+    first_seen_date,
+    normalized_channel,
+    normalized_os,
+    normalized_os_version,
+    locale,
+    country,
+    isp,
+    is_dau, --will work once the column is added by Kik
+    is_wau, --will work once the column is added by Kik
+    is_mau --will work once the column is added by Kik
+  FROM
+    `moz-fx-data-shared-prod.klar_ios.baseline_clients_last_seen`
+  WHERE
+    submission_date = @submission_date
+  UNION ALL
+  --Focus Android
+  SELECT
+    'Focus Android' AS source,
+    sample_id,
+    submission_date,
+    client_id,
+    first_seen_date,
+    normalized_channel,
+    normalized_os,
+    normalized_os_version,
+    locale,
+    country,
+    isp,
+    is_dau, --will work once the column is added by Kik
+    is_wau, --will work once the column is added by Kik
+    is_mau --will work once the column is added by Kik
+  FROM
+    `moz-fx-data-shared-prod.focus_android.baseline_clients_last_seen`
+  WHERE
+    submission_date = @submission_date
+  UNION ALL
+  --Focus iOS
+  SELECT
+    'Focus iOS' AS source,
+    sample_id,
+    submission_date,
+    client_id,
+    first_seen_date,
+    normalized_channel,
+    normalized_os,
+    normalized_os_version,
+    locale,
+    country,
+    isp,
+    is_dau, --will work once the column is added by Kik
+    is_wau, --will work once the column is added by Kik
+    is_mau --will work once the column is added by Kik
+  FROM
+    `moz-fx-data-shared-prod.focus_ios.baseline_clients_last_seen`
+  WHERE
+    submission_date = @submission_date
+),
+mobile_attribution AS (
+  --Fenix
+  SELECT
+    'Fenix' AS source,
+    client_id,
+    sample_id,
+    adjust_network,
+    adjust_campaign,
+    adjust_ad_group,
+    adjust_creative,
+    play_store_attribution_campaign,
+    play_store_attribution_source,
+    play_store_attribution_medium,
+    meta_attribution_app,
+    install_source,
+    NULL AS is_suspicious_device_client
+  FROM
+    `moz-fx-data-shared-prod.fenix_derived.firefox_android_clients_v1`
+  UNION ALL
+  --Firefox iOS
+  SELECT
+    'Firefox iOS' AS source,
+    client_id,
+    sample_id,
+    adjust_network,
+    adjust_campaign,
+    adjust_ad_group,
+    adjust_creative,
+    NULL AS play_store_attribution_campaign,
+    NULL AS play_store_attribution_source,
+    NULL AS play_store_attribution_medium,
+    NULL AS meta_attribution_app,
+    NULL AS install_source,
+    is_suspicious_device_client
+  FROM
+    `moz-fx-data-shared-prod.firefox_ios.firefox_ios_clients`
+)
+SELECT
```

Comment:
Missing a couple columns:
- app_name (to know which OS and also split BrowserStack)
- app_version (it's usually app_display_version on upstream tables, but rename for consistency with Desktop)

Reply:
done - added app_version